backend/src/models/Review.ts

```diff
@@ -6,6 +6,7 @@ import { ReviewKind, ReviewKindKeys } from '../types/enums.js'
 export const Decision = {
   RequestChanges: 'request_changes',
   Approve: 'approve',
+  UndoReview: 'undo_review',
```

Comment:
As approve review is just called `approve`, can we keep things consistent and refactor this to simply be `Undo: 'undo'`?